jax/_src/array.py

```diff
@@ -19,7 +19,7 @@
 import operator as op
 import numpy as np
 import functools
-from typing import Any, Callable, cast, TYPE_CHECKING
+from typing import Any, Enum, Callable, cast, TYPE_CHECKING
```

Comment:
This needs to come from the `enum` module:

```python
from enum import Enum
```

Reply:
Done.